source/elements/oneMKL/source/domains/blas/gemm_batch.rst

```diff
@@ -295,45 +316,83 @@ in ``a``, ``b`` and ``c`` are given by the ``batch_size`` parameter.
 
    namespace oneapi::mkl::blas::column_major {
        sycl::event gemm_batch(sycl::queue &queue,
-                              onemkl::transpose *transa,
-                              onemkl::transpose *transb,
-                              std::int64_t *m,
-                              std::int64_t *n,
-                              std::int64_t *k,
-                              T *alpha,
-                              const T **a,
-                              std::int64_t *lda,
-                              const T **b,
-                              std::int64_t *ldb,
-                              T *beta,
-                              T **c,
-                              std::int64_t *ldc,
-                              std::int64_t group_count,
-                              std::int64_t *group_size,
+                              T_transpose transa,
+                              T_transpose transb,
+                              T_dimension m,
```

Comment:
Can we group std::int64_t into a single template? I also would not use * in the template name. Let's keep */& in syntax

Reply:
Decided to share input parameters and write out each api syntax separately without template variables.  For clarity.